client/a8c-for-agencies/sections/signup/signup-v2/components/step-progress/style.scss

```diff
@@ -0,0 +1,73 @@
+.step-progress {
+	width: 100%;
+	background: var(--studio-white);
+
+	&__steps {
+		max-width: 600px;
+		margin: 0 auto;
+		width: 100%;
+		display: flex;
+		flex-direction: column;
+		gap: 16px;
+	}
+
+	&__steps-container {
+		display: grid;
+		grid-template-columns: repeat(3, 1fr);
+		gap: 8px;
+		width: 100%;
+	}
+
+	&__step {
+		display: flex;
+		flex-direction: column;
+		gap: 8px;
+
+		&.is-active {
+			.step-progress__step-indicator {
+				background-color: var(--studio-wordpress-blue);
+			}
+
+			.step-progress__step-label {
+				color: var(--studio-gray-80);
+				font-weight: 600;
+			}
+		}
+
+		&.is-complete {
+			.step-progress__step-indicator {
+				background-color: var(--studio-wordpress-blue);
+			}
+
+			.step-progress__step-label {
+				color: var(--studio-gray-80);
+				font-weight: 600;
+			}
+		}
+
+		&:not(.is-active):not(.is-complete) {
+			.step-progress__step-indicator {
+				background-color: var(--studio-gray-5);
```

Comment:
As I mentioned above. We can replace `--studio-gray-*` here with `--color-neutral-*`